File: DRRR.Client/src/app/user/dtos/user-login-request.dto.ts
/**
 * 用户登录请求DTO
 */
export interface UserLoginRequestDto {
  /**
   * 用户名
   */
  username?: string;

  /**
   * 密码
   */
  password?: string;

  /**
   * 是否是游客
   */
  isGuest: boolean;

  /**
   * 是否记住登录状态
   */
  rememberMe: boolean;
}

File: DRRR.Server/Dtos/UserLoginRequestDto.cs
namespace DRRR.Server.Dtos
{
    /// <summary>
    /// 用户登录请求DTO
    /// </summary>
    public class UserLoginRequestDto
    {
        /// <summary>
        /// 用户名
        /// </summary>
        public string Username { get; set; }

        /// <summary>
        /// 密码
        /// </summary>
        public string Password { get; set; }

        /// <summary>
        /// 是否是游客
        /// </summary>
        public bool IsGuest { get; set; }
    }
}
